Game of Game of Life.py

```diff
@@ -0,0 +1,98 @@
+### a,b refers to cell (a,b) whereas x,y refers to pixel coordinates ###
+
+import pygame as pg, math as m, time as t, random as r
+from config import *
+
+class Cell():
+    def __init__(self):
+        self.CurrentState=1
+        self.NextState=0
+    def Kill(self):
+        self.NextState=0
+    def Birth(self,Type):
+        self.NextState=Type
+
+def Check(a,b):
+    """Checks whether the cell will be dead or alive at the end of this turn,
+    and if so what type it will be"""
+    Type=Board[a][b].CurrentState
+    total=[0,0]
+    total[Board[a+1][b].CurrentState]+=1
+    total[Board[a][b+1].CurrentState]+=1
+    total[Board[a-1][b].CurrentState]+=1
+    total[Board[a][b-1].CurrentState]+=1
+    total[Board[a+1][b+1].CurrentState]+=1
+    total[Board[a-1][b-1].CurrentState]+=1
+    total[Board[a+1][b-1].CurrentState]+=1
+    total[Board[a-1][b+1].CurrentState]+=1
```

Comment:
Make this into a loop

Reply:
This will get in the way later on, when I (hopefully) introduce the new cell types - some of them don't take into account every square eg. only diagonals.